contracts/03-tamagotchi-nft/tests/owning.rs

```diff
@@ -66,7 +66,28 @@ fn interaction_test() {
 fn owning_test() {
     let sys = System::new();
     sys.init_logger();
-    let _program = Program::current(&sys);
+    let program = Program::current(&sys);
+    let _result = program.send(2, String::from("Goodmoring"));
+    let result = program.send(2, TmgAction::Transfer(1.into()));
+    let log = Log::builder()
+        .dest(2)
+        .payload(TmgEvent::Transferred(1.into()));
+    assert!(result.contains(&log));
+
+    // let result = program.send(1, TmgAction::Approve(2.into()));
```

Comment:
Need to check the transfer from approved and not approved accounts.

Reply:
Why am I unable to pass the test?
![Uploading telegram-cloud-photo-size-5-6055296394756994864-y.jpg…]()